### PR TITLE
 Update automatically correct image version and revert broken version for master

### PR DESCRIPTION
**Pull request by FilipB**

### CODE REVIEW

File: ci-operator/config/maistra/test-infra/maistra-test-infra-main.yaml [hunk 1]
```diff
@@ -90,15 +90,17 @@ tests:
...
     ./tools/automator-main.sh \
     --org=openshift \
     --repo=release \
     --branch=master \
     --token-path=/creds-github/token \
     '--title=Update maistra/istio builder image' \
     --modifier=update_istio_builder_image_master \
-    '--cmd=sed -i "s|\(gcr.io/istio-testing/build-tools\):.*|\1:$NEW_IMAGE|" core-services/image-mirroring/_config.yaml' \
+    '--cmd=sed -i "s|\(gcr.io/istio-testing/build-tools\):master.*|\1:$NEW_IMAGE_MASTER|" core-services/image-mirroring/_config.yaml' \
+    '--cmd=sed -i "s|\(gcr.io/istio-testing/build-tools\):release-1\.22.*|\1:$NEW_IMAGE_1_22|" core-services/image-mirroring/_config.yaml' \
```
Comment (jwendell) [inline ci-operator/config/maistra/test-infra/maistra-test-infra-main.yaml:103]: will this work? two `--cmd` flags? Will it ignore the first and only consider the last one?

I think you will need to merge the two sed commands into a single sed.

Comment (jwendell) [reply ci-operator/config/maistra/test-infra/maistra-test-infra-main.yaml:103]: or two seds separated by `&&` i.e., one single `--cmd` flag that runs multiple commands separated by `&&`

Comment (FilipB) [reply ci-operator/config/maistra/test-infra/maistra-test-infra-main.yaml:103]: Not sure, better to use just one. Fixing. Thanks

**Review by jwendell — APPROVED** [review]

**Pull request merged**